### DISCUSSION TITLE
DIAMOND theme

### DISCUSSION THREAD

**Discussion by cvexa** [primefaces/community#516] [PrimeReact Templates]
Hello, 

just bought https://diamond.primereact.org/ 
and wanted to replace the existing theme of a project with this diamond one, but all files are .tsx and my current app is using JSX, so how can I use components from the theme in my JSX app? Tried with some online convertors but still cannot make it work, for example, the sidebar....
Is there any way to get the theme with jsx files, so I can directly use them in my existing app?

**Comment by melloware** [comment]
@cvexa you should be able to use both TSX and JSX files in the same app.  You don't need to have all typescript or not you can use indvidual files from both JSX or TSX.

**Comment by mertsincan** [comment]
Hi, 

I'm sorry for not getting back to you sooner! Unfortunately, Diamond doesn't have JS version.  we can also expect upvotes from other users for Diamond-JSX version. We will be happy to implement this when there are enough upvotes. Thanks a lot for your understanding!
